app/helpers/application_helper.rb

```diff
@@ -68,6 +68,16 @@ def class_for_alert_colour(colour)
     end
   end
 
+  def temperature_cell_colour(temperature)
+    if temperature >= 19
+      'table-danger'
+    elsif temperature < 18.0
```

Comment:
Does the `.0` do something different to the 19 that doesn't have one?

Reply:
No, what the heck was that there for? Will tweak